packages/core/components/ExternalInput/index.tsx

```diff
@@ -100,13 +113,44 @@ export const ExternalInput = ({
           className={getClassNameModal({
             isLoading,
             loaded: !isLoading,
-            hasData: !!data,
+            hasData: data.length > 0,
           })}
         >
           <div className={getClassNameModal("masthead")}>
             <Heading rank={2} size="xxl">
               Select content
             </Heading>
+
+            {field.showSearch && (
+              <form
+                className={getClassNameModal("searchForm")}
+                onSubmit={(e) => {
+                  e.preventDefault();
+
+                  search(searchQuery);
+                }}
+              >
+                <label className={getClassNameModal("search")}>
+                  <span style={{ visibility: "hidden", width: 0 }}>Search</span>
```

Comment:
This will be hidden from assistive technology, and neither thew feather icon of the placeholder will provide an accessible alternative.

Best approach would be to use this CSS snippet here instead, which would make the label text inclusively available.

```
  clip: rect(0 0 0 0);
  clip-path: inset(100%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap;
  width: 1px;
```